solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/FeatureStore.java

```diff
@@ -19,27 +19,26 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 
 import org.apache.solr.ltr.ranking.Feature;
-import org.apache.solr.ltr.util.FeatureException;
 
 public class FeatureStore {
-  LinkedHashMap<String,Feature> store = new LinkedHashMap<>();
-  String storeName;
+  private final Map<String,Feature> store = new LinkedHashMap<>(); // LinkedHashMap because we need predictable iteration order
```

Comment:
You should keep this as a LinkedHashMap since we are dependent on this specific type of map due to the predictable ordering.  Using Map implies this could work with any other map, which isn't true.